dns-hosting.md

```diff
@@ -0,0 +1,20 @@
+---
+title: DNS hosting
+layout: recommendation
+expires: 2018-01-01
+---
+To make services available on the internet they need to have DNS records to let people know where they are.
+service.gov.uk domains are delegated to [nameservers](https://www.gov.uk/service-manual/technology/get-a-domain-name#check-where-youll-host-your-dns) controlled by service teams.
+
+
+## Principles
+
+- DNS entries should be version controlled unless autogenerated
+- Nameservers should be redundant enough for your service.
```

Comment:
Within that context that sentence makes sense. I think it's the "enough" in that sentence which makes it vague. If we recommended that "Nameservers should be sufficiently redundant for your service requirements" that would probably work.